Symfony/CS/Tests/UtilsTest.php

```diff
@@ -17,21 +17,24 @@
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author Graham Campbell <graham@mineuk.com>
+ * @author Odín del Río <odin.drp@gmail.com>
  */
 class UtilsTest extends \PHPUnit_Framework_TestCase
 {
     /**
      * @dataProvider provideCamelCaseToUnderscoreCases
+     *
+     * @param string $expected Camel case string.
+     * @param string $input    Input string.
      */
-    public function testCamelCaseToUnderscore($expected, $input = null)
+    public function testCamelCaseToUnderscore($expected, $input)
```

Comment:
that behavior was intended.
now it wont test if 'foo_bar' wasn't modified.

please revert that.